File: step_by_step/step_5/c/1546.c
#include <stdio.h>

int main(void){
    int N;
    scanf("%d", &N);

    int arr[N];
    int max = 0;
    for(int i=0; i<N; i++){
        scanf("%d", &arr[i]);
        if(arr[i] > max)
            max = arr[i];
    }

    double sum = 0; // double 주의
    for(int j=0; j<N; j++){
        sum = sum + ((double) arr[j]/max)*100; // double 주의
    }
    printf("%f\n", (double) sum/N); // %f, double 주의
}

File: step_by_step/step_5/python/1546.py
# 평균

N = int(input())
scores = list(map(int, input().split()))
M = max(scores)
# map을 이용해 list 내의 각 item에 함수(lambda) 적용
new_scores = list(map(lambda x: (x/M)*100, scores))
# print(new_scores)
print(sum(new_scores) / len(new_scores))